src/app/app.js

```diff
@@ -89,11 +91,6 @@ angular.module('prx', ['ngAnimate',
         type: "item"
       },
     ]
-  }, {
-    name: 'Sign In',
-    weight: PRXDrawerProvider.BOTTOM,
-    href: 'http://www.prx.org/sessions/new',
-    nav: true
```

Comment:
We're removing the old sign-in link because it's added to the nav (in a different form) elsewhere. This was part of the goal of setting up the nav this way, so that each module could register its own stuff to it and we wouldn't need to have knowledge about individual modules (e.g. the Auth module) spread too far outside of themselves.